Lib/test/test_posix.py

```diff
@@ -1220,6 +1227,7 @@ def test_sched_setaffinity(self):
             self.assertRaises(OSError, posix.sched_setaffinity, -1, mask)
 
     @unittest.skipIf(support.is_wasi, "No dynamic linking on WASI")
+    @unittest.skipUnless(os.name == 'posix', "requires Posix")
```

Comment:
The test is called "posix" and other tests are run. It's surprising that a test is skipped because it "requires Posix".

Maybe skip the test on Windows, is support.MS_WINDOWS is true? With a message like "Windows doesn't have RTLD constants".

Reply:
The message for the skip test as is would better be "requires posix module".  But I think Victor's suggest is even better.  It there were a 3rd OS module, we don't know that the test would need to be skipped.

Reply:
I did not invent this message, it is one of common messages already used in many other tests with this condition. Grep `'skip.*posix'`.

Reply:
Windows provides a POSIX-like API for many years. You say that Linux is not Unix. Well, I would still prefer to say "Windows doesn't have RTLD constants" than just "requires Posix". By the way, POSIX is an accronym which stands for "Portable Operating System Interface", and so should be written in uppercase: https://en.wikipedia.org/wiki/POSIX ;-)

Python is misleading since `os.name` is just ``"posix"`` on all platforms but Windows (where it's ``"nt"``): https://pythondev.readthedocs.io/platforms.html#sys-platform-versus-os-name Is it the name of the operating system ("OS") or the name of the C extension? The "os" name is not called "os?" :-)

Reply:
I must have confused him with FORTRAN that is now simply Fortran.